### ISSUE TITLE
Why are the reference image / target images name flipped?

### ISSUE THREAD

**Issue by Cerfy** [sunghoonim/DPSNet#19]
Hi, I know this is a trivial question but would just like to clarify the train_loader and dpsnet passed in parameters as I noticed the variable names are different. Why isn't the naming conventions consistent? 

Loading in the data:
`for i, (tgt_img, ref_imgs, ref_poses, intrinsics, intrinsics_inv, tgt_depth) in enumerate(train_loader):`

Doing a forward pass:
`depths = dpsnet(tgt_img_var, ref_imgs_var, pose, intrinsics_var, intrinsics_inv_var)`

In PSNet Class:
`def forward(self, ref, targets, pose, intrinsics, intrinsics_inv):`

**Comment by MASTERCHIEF343** [comment]
yeah, i also notice that.But in the paper the writer says that he use an input image pair for cost volume construction.So i print the lenght of targets(in PSNet Class) and it's one.May be it doesn't have an influence.This is what i think.
